modules/ROOT/pages/user-task-list.adoc

```diff
@@ -1,13 +1,14 @@
 = User task list
-:description: This page explains what a user with the User profile in Bonita Portal or in xref:user-application-overview.adoc[Bonita User Application] can see and do about tasks. +
+:description: This page explains what a user can see and do about tasks in the xref:user-application-overview.adoc[Bonita User Application]. +
 
-This page explains what a user with the User profile in Bonita Portal or in xref:user-application-overview.adoc[Bonita User Application] can see and do about tasks. +
+
+{description} +
 _Users_ can view lists of tasks to do as a member of a team, view tasks precisely assigned to them, view tasks they have done related to different processes, take and release tasks, and do tasks, eventually.
```

Comment:
_Users_ can view lists of tasks to do as a member of a team, view tasks precisely assigned to them, view tasks they have done related to different processes, take and release tasks, and do tasks, eventually. => _Users_ can view lists of tasks assigned to their team, assign a task to them, view lists of tasks assigned to them inside a team, work on the tasks, view tasks the "Done" tasks on different processes, release tasks (depending on their status).

Reply:
Added